sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/SwitchSectionShouldNotHaveTooManyStatements_DefaultValue.vb

```diff
@@ -0,0 +1,48 @@
+﻿Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+Imports System.Text
+Imports Microsoft.VisualBasic
+
+Namespace Tests.TestCases
+    Class Foo
+        Public Sub Test()
+            Dim number As Integer = 8
+
+            Select Case number
+                Case 1 To 5 ' Noncompliant {{Reduce this 'Case' block number of statements from 9 to at most 8, for example by extracting code into a procedure.}}
+'               ^^^^^^^^^^^
+                    Test()
+                    Test()
+                    Test()
+                    Test()
+                    Test()
+                    Test()
+                    Test()
+                    Test()
+                    Test()
+                Case 6 ' Noncompliant {{Reduce this 'Case' block number of statements from 12 to at most 8, for example by extracting code into a procedure.}}
+                    If False
```

Comment:
You should add coverage for the following as well
                    case SyntaxKind.CatchBlock:
                    case SyntaxKind.FinallyBlock:
                    case SyntaxKind.ForEachBlock:
                    case SyntaxKind.SyncLockBlock:
                    case SyntaxKind.TryBlock:
                    case SyntaxKind.UsingBlock:
                    case SyntaxKind.WithBlock:
                    case SyntaxKind.EndSyncLockStatement:
                    case SyntaxKind.EndTryStatement:
                    case SyntaxKind.EndUsingStatement:
                    case SyntaxKind.EndWithStatement: